README.md

```diff
@@ -1154,6 +1157,127 @@ supported for various relational database flavors:
 | SQLite     | Root-only queries (no edges), [@output](#output), [@filter](#filter) | [\__typename](#__typename) output metafield, [intersects](#intersects) filter,  [has_edge_degree](#has_edge_degree) filter and [name_or_alias](#name_or_alias) filter unsupported|
 
 
+### Configuring SQLAlchemy
+Support for relational databases is accomplished by compiling to SQLAlchemy core as an intermediate
```

Comment:
nit: a lot of sentences here use passive voice, which makes them a bit harder to read